### PR TITLE
Pause doc: Item page quads

### PR DESCRIPTION
**Pull request by Dragorn421**

### CODE REVIEW

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h [hunk 1]
```diff
@@ -110,6 +110,63 @@ typedef enum PromptQuad {
...
+    /* 33 */ ITEM_QUAD_AMMO_BOW_TENS,
+    /* 34 */ ITEM_QUAD_AMMO_BOW_UNITS,
+    /* 35 */ ITEM_QUAD_AMMO_SLINGSHOT_TENS,
+    /* 36 */ ITEM_QUAD_AMMO_SLINGSHOT_UNITS,
+    /* 37 */ ITEM_QUAD_AMMO_BOMBCHU_TENS,
+    /* 38 */ ITEM_QUAD_AMMO_BOMBCHU_UNITS,
+    /* 39 */ ITEM_QUAD_AMMO_BEAN_TENS,
+    /* 40 */ ITEM_QUAD_AMMO_BEAN_UNITS,
+    /* 41 */ ITEM_QUAD_AMMO_MAX,
+    /* 40 */ ITEM_QUAD_AMMO_LAST = ITEM_QUAD_AMMO_MAX - 1,
```
Comment (cadmic) [inline src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h:166]: having both `LAST` and `MAX` seems a bit weird IMO. And it seems both `ITEM_QUAD_AMMO_MAX` and `ITEM_QUAD_AMMO_LAST` are unused anyway?

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h [hunk 2]
```diff
@@ -110,6 +110,63 @@ typedef enum PromptQuad {
...
+    /*  0 */ ITEM_QUAD_GRID_FIRST,
+    /* 23 */ ITEM_QUAD_GRID_LAST = ITEM_GRID_ROWS * ITEM_GRID_COLS - 1,
+    // Markers indicating the currently equipped items
+    /* 24 */ ITEM_QUAD_GRID_SELECTED_C_LEFT,
+    /* 25 */ ITEM_QUAD_GRID_SELECTED_C_DOWN,
+    /* 26 */ ITEM_QUAD_GRID_SELECTED_C_RIGHT,
+    // Digits for showing ammo count
+    /* 27 */ ITEM_QUAD_AMMO_FIRST,
+    /* 27 */ ITEM_QUAD_AMMO_STICK_TENS = ITEM_QUAD_AMMO_FIRST,
+    /* 28 */ ITEM_QUAD_AMMO_STICK_UNITS,
```
Comment (cadmic) [inline src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h:152]: Usually this is called the "ones digit", so `ITEM_QUAD_AMMO_STICK_ONES` (ditto for `ITEM_AMMO_UNITS_QUAD_OFFSET_X` and the comments in the drawing code)

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h [hunk 3]
```diff
@@ -110,6 +110,63 @@ typedef enum PromptQuad {
...
+// Relative to the corresponding ITEM_QUAD_GRID_ quad
+#define ITEM_AMMO_TENS_QUAD_OFFSET_X 0
+#define ITEM_AMMO_TENS_QUAD_OFFSET_Y 22
+
+// Relative to the ammo tens digit
+#define ITEM_AMMO_UNITS_QUAD_OFFSET_X 6
+#define ITEM_AMMO_UNITS_QUAD_OFFSET_Y 0
+
+typedef enum ItemQuad {
+    // 0 to 23 are the ITEM_GRID_ROWS*ITEM_GRID_COLS item grid
```
Comment (cadmic) [inline src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h:141]: it it worth giving a name to ITEM_GRID_ROWS*ITEM_GRID_COLS?

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h (hunk 1 above)
Comment (Dragorn421) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h:166]: removed first, last and max, not sure why I added them in the first place

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h (hunk 3 above)
Comment (Dragorn421) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h:141]: it's only used one time besides the two spots in this enum, so imo no?

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c [hunk 4]
```diff
@@ -2984,34 +3002,54 @@ void KaleidoScope_SetVertices(PlayState* play, GraphicsContext* gfxCtx) {
...
-        pauseCtx->itemVtx[i + 0].v.ob[0] = pauseCtx->itemVtx[i + 2].v.ob[0] = pauseCtx->itemVtx[k].v.ob[0];
+        pauseCtx->itemVtx[i + 0].v.ob[1] = pauseCtx->itemVtx[i + 1].v.ob[1] =
+            pauseCtx->itemVtx[k].v.ob[1] - ITEM_AMMO_TENS_QUAD_OFFSET_Y;
 
-        pauseCtx->itemVtx[i + 1].v.ob[0] = pauseCtx->itemVtx[i + 3].v.ob[0] = pauseCtx->itemVtx[i + 0].v.ob[0] + 8;
+        pauseCtx->itemVtx[i + 2].v.ob[1] = pauseCtx->itemVtx[i + 3].v.ob[1] =
+            pauseCtx->itemVtx[i + 0].v.ob[1] - ITEM_AMMO_DIGIT_QUAD_HEIGHT;
 
-        pauseCtx->itemVtx[i + 0].v.ob[1] = pauseCtx->itemVtx[i + 1].v.ob[1] = pauseCtx->itemVtx[k].v.ob[1] - 22;
+        // units
```
Comment (cadmic) [inline src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c:3038]: "ones" (ditto below)

**Review by cadmic — APPROVED** [review]

**Review by fig02 — APPROVED** [review]

**Pull request merged**